src/assets/translations/fr/main.json

```
@@ -2772,6 +2788,101 @@
     "pendingDistribution": "Distribution en attente",
     "selectRuneAddress": "Sélectionnez une adresse RUNE"
   },
+  "TCY": {
+    "stake": "Miser",
+    "unstake": "Démiser",
+    "staking": "Mises TCY",
+    "myPosition": "Ma position",
+    "myStakedBalance": "Mon solde misé",
+    "myStakedBalanceHelper": "Le montant en %{symbol} que vous avez misé",
+    "pendingRewardsBalance": "Solde des récompenses en attente",
+    "pendingRewardsBalanceHelper": "Il s'agit seulement d'une estimation de vos récompenses en attente, basée sur vos revenus cumulés à ce jour. Toute la comptabilité est effectuée à la fin de l'époque et ne peut être calculée avec précision avant la fin du mois.",
+    "lifetimeRewards": "Récompenses cumulées",
+    "lifetimeRewardsHelper": "Le total des récompenses que vous avez gagnées à ce jour en misant vos %{symbol}",
+    "timeStaked": "Durée de mise",
+    "timeStakedHelper": "Le temps pendant lequel vous avez misé vos %{symbol}",
+    "stakingReward": "Récompense de mise",
+    "claimsEmpty": {
+      "emptyTitle": "Aucune réclamation disponible",
+      "emptyBody": "Il semble que vous n'ayez aucune réclamation TCY disponible. Essayez avec un autre porte-monnaie ?",
+      "connectWalletBody": "Connectez votre porte-monnaie pour voir vos réclamations TCY."
+    },
+    "claimSelect": {
+      "unavailableClaims": "Réclamations indisponibles",
+      "notEligible": "Non éligible"
```

ocd: https://www.dictionnaire-academie.fr/article/A9N0575

> ▪  Uni par un trait d’union à des noms, à des adjectifs, etc., non entre dans la formation de mots composés, souvent empruntés de l’anglais, dont les plus courants figurent ci-dessous à leur ordre alphabétique (Non ne prendra jamais la marque du pluriel).

```suggestion
      "notEligible": "Non-éligible"
```

---

Such a *complex* topic that it has [its own page on Wikipedia](https://fr.wikipedia.org/wiki/Emploi_du_trait_d%27union_pour_les_pr%C3%A9fixes_en_fran%C3%A7ais#Adverbes_non,_hors,_tr%C3%A8s_et_quasi) actually:

> Autrement, si le mot suivant est un [adjectif](https://fr.wikipedia.org/wiki/Adjectif), alors on n'utilise pas de trait d'union après l'adverbe, par exemple une facture non payée, quasi certain, un arbre très haut.

With this exception:

> Concernant l'adverbe non, plus on considère telle ou telle construction de type non + adjectif ou participe passé comme lexicalisée, c’est-à-dire comme un véritable mot composé, et plus on a tendance à l’écrire avec un trait d’union. 

But I don't think "non eligible" is that lexicalized... I can't find dictionaries that list it with a *trait d'union*.